lib/fastlane/actions/mailgun.rb

```diff
@@ -0,0 +1,136 @@
+module Fastlane
+  module Actions
+    class MailgunAction < Action
+      def self.git_author
+        s = `git log --name-status HEAD^..HEAD`
+        s = s.match(/Author:.*<(.*)>/)[1]
+        return s if s.to_s.length > 0
+        return nil
+      rescue
+        return nil
+      end
+
+      def self.last_git_commit
+        s = `git log -1 --pretty=%B`.strip
+        return s if s.to_s.length > 0
+        nil
+      end
+
+      def self.is_supported?(platform)
+        true
+      end
+
+      # As there is a text limit in the notifications, we are
+      # usually interested in the last part of the message
+      # e.g. for tests
+      def self.trim_message(message)
```

Comment:
There is no text limit for emails, do we still want to trim the message? I think it should at least be possible to disable it. What do you think?